src/api/app/assets/stylesheets/webui/build-result.scss

```diff
@@ -103,3 +166,14 @@
 .repository-state-published {
   @extend .text-success;
 }
+
+@include media-breakpoint-up(md) {
+  .build-info .triangle.left { left: .25rem; }
+}
+
+@include media-breakpoint-between(sm, md) {
+  .build-info .triangle {
+    &.center { margin-left: calc(38% + .75rem); }
+    &.left { left: .5rem; }
+  }
+}
```

Comment:
[Newline missing at the end of file](https://thoughtbot.com/blog/no-newline-at-end-of-file)